explainaboard/metrics/nlg_meta_evaluation.py

```diff
@@ -18,6 +19,142 @@
 from explainaboard.utils.typing_utils import narrow, unwrap_or
 
 
+@dataclass
+@common_registry.register("CorrelationNLGConfig")
+class CorrelationNLGConfig(MetricConfig):
+    """Configuration of a correlation for general NLG tasks.
+
+    Args:
+        level: there are following levels: sample level, system level and dataset level
+                See more details: https://aclanthology.org/2020.emnlp-main.751.pdf
```

Comment:
I think it is better to be renamed this to *strategy* as referred in the section 2.5 of the paper. "level" seems somewhat confusing because we have already AnalysisLevels.

Reply:
Sounds good. I will follow this suggestion.